workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+---
+name: 'Konami Code'
+description: 'Implement a konami code on a website'
+author: '@faisalsayed10'
+---
+
+# Konami Code
+
+You may have heard about movie Easter Eggs or video game Easter Eggs before, but have you ever heard about website Easter Eggs? The internet is filled with copious amounts of weird and wonderful hidden features! I bet you have played the popular game GTA, right? GTA too had konami codes but they were called 'cheat codes'. 
+
+If you don't know what they are, check out these cool easter eggs/konami codes on the internet:
+
+1. Try searching 'do a barrel roll' on google.
+
+2. Try inspecting [Max's Website](maxwofford.com) 🤨
+
+3. Konami Code On [BuzzFeed](https://www.buzzfeed.com/) (Just press up, up, down, down, left, right, left, right, B, A and enter on the homepage).
+
+4. In the Firefox browser bar, type "about:mozilla" or "about:robots" to see the secrets of Mozilla.
+
+Well, by now, you must have known what a konami code is. How about we build our own konami code?
+
+It will look something like this after someone types the konami code:
+
+[![Image](https://cloud-5apj3kt1e.vercel.app/image.png)](https://konami-code.faisalsayed1.repl.co/)
+
+Here's the [live demo](https://konami-code.faisalsayed1.repl.co/)  
+Here's the [source code](https://repl.it/@FaisalSayed1/Konami-Code)
+
+Live demo: 
+![Image](https://cloud-95l2730mf.vercel.app/konamigif.gif)
+
+## Part 1: Prerequisites
+
+You should have a beginner understanding of:
+
+- HTML
+- CSS
+- JavaScript
+
+Also for this workshop, we will need a library called 'confetti.js'. 
+
+Download it by going [here](https://cloud-72a7nfmm7.vercel.app/index.min.js). Right click and select 'save as'.
+
+## Part 2: Setup
+
+### Setting up your code environment on Repl.it
+
+[Repl.it](https://repl.it) is an online code editor. You don't have to use Repl.it but I suggest you do as it sets everything up for you and you don't require any installations.
+
+To get started, go to [https://repl.it/languages/html](https://repl.it/languages/html). Your coding environment will spin up in just a few seconds!
+
+You should see something like the following:
+
+![Image](https://cloud-p7qnbqzo6.vercel.app/image.png)
+
+Import the confetti.js (`index.min.js`) file which you downloaded on the left side of the screen.
+
+![Image](https://cloud-2d72c0n78.vercel.app/indexminjs.gif)
+
+## Part 3: Building the project
+
+### 1) HTML
+
+Let's start writing the HTML code required in the `index.html` file. First, we'll import the required JavaScript files.
+
+Inside the `<head>` tag, we'll import the [`anime.js`](https://animejs.com/) library. Copy the code given below.
+
+```html
+<script src="https://cutt.ly/anime_js"></script>
+```
+
+
+Also link the `index.min.js` file to our HTML inside the `<head>` tag.
+
+```html
+<script src="index.min.js"></script>
+```
+
+The code so far:
+
```

Comment:
```suggestion

Let's start writing the HTML code required in the `index.html` file. First, we'll import the required JavaScript files.

Inside the `<head>` tag, we'll import the [`anime.js`](https://animejs.com/) library. Copy the code given below.

```html
<script src="https://cutt.ly/anime_js"></script>
```


Also, link the `index.min.js` file to our HTML inside the `<head>` tag.

```html
<script src="index.min.js"></script>
```

The code so far:

```

Reply:
To be honest, you have confused me so much

Reply:
oh sorry
i just fixed the typos